Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -341,6 +369,28 @@ private void ExecuteResetResultsCommand(object parameter)
             Tests.Refresh();
         }
 
+        private void ExecuteIgnoreTestCommand(object parameter)
+        {
+            var rewriteSession = RewritingManager.CheckOutCodePaneSession();
+            var qualifiedContext = new QualifiedContext(_mousedOverTestMethod.Declaration.QualifiedModuleName, _mousedOverTestMethod.Declaration.Context);
+
+            var annotationUpdater = new AnnotationUpdater();
+            annotationUpdater.AddAnnotation(rewriteSession, qualifiedContext, Parsing.Annotations.AnnotationType.IgnoreTest);
+
+            rewriteSession.TryRewrite();
+        }
+
+        private void ExecuteUnignoreTestCommand(object parameter)
+        {
+            var rewriteSession = RewritingManager.CheckOutCodePaneSession();
+            var ignoreTestAnnotation = _mousedOverTestMethod.Declaration.Annotations.First(iannotation => iannotation.AnnotationType == Parsing.Annotations.AnnotationType.IgnoreTest);
```

Comment:
would it not be necessary to mark "all" the annotations of that type for removal?

Reply:
I would also say that removing all seems to be more sensible. This will just require replacing `First` with `Where` and adding an `s` to the variable name.